src/data/init/init-abilities.ts

```diff
@@ -202,6 +203,12 @@ import { TerrainType } from "#enums/terrain-type";
 import { WeatherType } from "#enums/weather-type";
 import i18next from "i18next";
 
+/** Used for Aerialate, Refrigerate, Pixilate, Galvanize */
+const basicMoveTypeChangeAbilityCondition: PokemonAttackCondition = (user, _target, move) =>
```

Comment:
A better name for this is welcome.

Reply:
So they all alter normal-type moves to something else.
- https://bulbapedia.bulbagarden.net/wiki/Galvanize_(Ability)
- https://bulbapedia.bulbagarden.net/wiki/Aerilate_(Ability)
- https://bulbapedia.bulbagarden.net/wiki/Pixilate_(Ability)
- https://bulbapedia.bulbagarden.net/wiki/Refrigerate_(Ability)

maybe 
```suggestion
const alterNormalTypeMoveCondition: PokemonAttackCondition = (user, _target, move) =>
```

Reply:
I've heard these abilities referred to as -ate abilities online but I don't know if that would be a good name. Also for reference, the only abilities that change move types are Aerilate/Galvanize/Pixilate/Refrigerate which change normal->flying/electric/fairy/ice and give a 20% boost, normalize (change everything to normal and give a 20% boost), and liquid voice (changes sound moves to water with no atk boost)

Reply:
https://pokemondb.net/pokebase/244961/what-are-all-of-the-ate-abilitites